commands/UpdateCommand.js

```
@@ -18,7 +18,7 @@ export class UpdateCommand extends Commander {
     return {
       ok: true,
       wip: updated,
-      message: `Issue #${id} update draft saved`,
+      display: `Issue #${id} update draft saved`,
```

Biome (2.4.15)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)